### DISCUSSION TITLE
Bella ciao multi language music video

### DISCUSSION THREAD

**Discussion by Azadi2022** [WeAreMahsaAmini/Core#42] [Ideas]
I googled and found English, Spanish, French and  Dutch version of the song.(maybe we can find/add Arabic version too?)The original song is Italian and I’m sure you guys listen to the beautiful Farsi version. 

Imagine 6 singers composed this revolutionary song and sing it in their own language. How impressive!

**Comment by anonymooshia** [comment]
If someone can help pull together the relevant content (music + images), I can help with the video editing and audio mixing.

**Reply by shirinrs** [reply]
Hi, I am a soprano singer and I speak Farsi, English, and Italian - maybe we can have our own version?
I have other friends who sing too :)

**Reply by anonymooshia** [reply]
That could work too. Are you thinking you'll be sharing your face in the video too or mostly interested in being the voice(s)? Either way can work I suppose.

**Reply by Azadi2022** [reply]
thanks @anonymooshia, we definitely can use your editing skills 🙏🏻

**Reply by Azadi2022** [reply]
@shirinrs yeah of course. please let us know if you know other talented artists who can help us with other languages.

**Reply by ispooya** [reply]
Hi, one of my friends is a creative producer and video creator who can help us a lot

**Reply by ghost** [reply]
My cousin, speaks french, English and Farsi. He can help in any language.

**Comment by sinahnyazdi** [comment]
I can sing in Farsi and English, rock/metal style

**Reply by Azadi2022** [reply]
Thanks Sina jaan, we can also use rock/metal for sure 🙏🏻

**Comment by Setareh4IRAN** [comment]
I may be able to play a violin version if needed

**Reply by Azadi2022** [reply]
Perfect Setareh jaan, Thanks🙏🏻

**Comment by shirinrs** [comment]
Hello everyone, does anyone know a composer/mixer? so we can start

**Comment by Azadi2022** [comment]
Hi everyone. I'm preparing an email to send to music schools around the world to find more talented artist for this project. At the same time I'm looking for a skilled composer.
I guess the most doable thing is to gather artists online and make the music video over for example google meet. We've seen the same approach earlier at the beginning of Covid. 
Please let us know if you have any suggestion or you can recommend someone. 
If you wanna help me to send the email requests please mention me under this post.

**Reply by Setareh4IRAN** [reply]
Madgal (singer), Emma (singer, song writer), Kia rokni (Beatboxer+mix and mastering) .. I don't know if they can cooperate or not. But if they do, it will be good :)

**Reply by ghost** [reply]
I know a musician in Toronto who might be able to help.

**Comment by Azadi2022** [comment]
Here is the 2 emails templates I prepared. Please spread them and make our project go viral.

First one is a call for artists 9(We can send it to music schools for example):

Dear Sir or Madam

We are a group of international singers and players who want to create a music video to help Iranian protestors spreading their liberty message around the world. Our purpose is to sing the famous “Bella Ciao”  by  Manu Pilas in Farsi, Italian, Spanish, French, Dutch and Arabic.  We wanted to ask  if your students and artists are interested to participate in our project.

Thank you so much for supporting democracy in Iran

**Comment by Azadi2022** [comment]
This one is to send to composers. Please send this one to famous composers for more credibility.

Dear …

Thank you so much for supporting the Iranian movement in this critical moment in the history of the country. We are a group of international singers and players who want to create a music video to help Iranian protestors spreading their liberty message around the world. Our purpose is to sing the famous “Bella Ciao”  by  Manu Pilas in Farsi, Italian, Spanish, French, Dutch and Arabic.  We wanted to ask  if we can use your talented mind  to compose a master piece for us.

Thank you so much for supporting democracy in Iran

**Reply by shirinrs** [reply]
Great thanks for email template, I am reaching out to some composers I know- will keep you posted

**Comment by mostafa-saeadi** [comment]
https://open.spotify.com/album/3qALxrUKi0oSirTOajohTL?si=IAF4IdhLS72vnGVbsEncdg&utm_source=copy-link

this can be a good persian version of this song
